### PR TITLE
v1.8 backports 2020-11-09

### PR DESCRIPTION
**Pull request by tklauser**
* #13892 -- release: add script to check presence of docker images (@aanm)
 * #13927 -- test/bpf: Fix XDP loading in verifier-test.sh (@pchaigno)
 * #13907 -- operator: increase GC Rate limit of identities to 2500 per minute (@aanm)
 * #13886 -- fqdn: Add a nil check for security id lookup (@aditighag)
 * #13914 -- fqdn: keep IPs alive if their name is alive (@kkourt)
 * #13921 -- bpf: fix disable PolicyVerdictNotification broken (@ArthurChiao)
 * #13922 -- bpf: Compile test POLICY_VERDICT_NOTIFY (@pchaigno)
   - Merge conflict in LXC_OPTIONS in bpf/Makefile.
 * #13878 -- gke: add resource quotas for Cilium Namespace (@aanm)
   - Merge conflict due to helm refactoring.
 * #12658 -- test: Move RuntimeVerifier to K8sVerifier (@pchaigno)
   - Minor merge conflict in test/helpers/manifest.go
* #12865 -- Hubble-Relay: proxy context from originating client (@nathanjsweet)
* #13206 -- Fix GetFlows Test (@nathanjsweet)

Once this PR is merged, you can update the PR labels via:
```upstream-prs
$ for pr in 13892 13927 13907 13886 13914 13921 13922 13878 12658 12865 13206; do contrib/backporting/set-labels.py $pr done 1.8; done
```

Dropped:

 * #13908 -- bpf: don't access TCP flags for non initial IPv4 fragments (@jibi)
   - Hit complexity issue, see https://github.com/cilium/cilium/pull/13951#issuecomment-724729806

### CODE REVIEW

**Comment by tklauser** [comment]
test-backport-1.8

**Comment by tklauser** [comment]
test-backport-1.8

**Review by pchaigno — APPROVED** [review]
My PRs look good :+1:

**Comment by tklauser** [comment]
test-backport-1.8

File: test/k8sT/manifests/test-verifier.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-verifier
+spec:
+  containers:
+  - name: cilium-builder
+    image: quay.io/cilium/cilium-builder:2020-10-20
```
Comment (pchaigno) [inline test/k8sT/manifests/test-verifier.yaml]: Maybe we should switch this to use `quay.io/cilium/cilium-builder:2020-10-15-v1.8@sha256:5f2e2e4cb4b86525f23cc183ffabb0297060b269f830377a548ea60c058951f9`, i.e., the tag we use in v1.8?

Comment (tklauser) [reply test/k8sT/manifests/test-verifier.yaml]: Good point, done (and updated to point to the newly built `quay.io/cilium/cilium-builder:2020-11-09-v1.8@sha256:660cb6ec90c0e662b710a86ab876ecf1a1d0a503269c6751ef3d2c4afc18666e` tag)

**Comment by tklauser** [comment]
test-backport-1.8

**Comment by tklauser** [comment]
test-missed-k8s

**Review by aanm — APPROVED** [review]
LGTM for my commits

**Comment by tklauser** [comment]
Previous failure was due to a complexity issue, likely introduced by backporting #13908 (thanks @pchaigno for investigating), see https://cilium.slack.com/archives/C7PE7V806/p1605016684205600 for discussion.

Will drop #13908 from this PR.

**Comment by tklauser** [comment]
test-backport-1.8

**Review by borkmann — APPROVED** [review]

**Pull request merged**